plugins/extractors/bigquery/bigquery.go

```diff
@@ -30,7 +32,9 @@ var summary string
 
 // Config holds the set of configuration for the bigquery extractor
 type Config struct {
-	ProjectID            string   `mapstructure:"project_id" validate:"required"`
+	ProjectID string `mapstructure:"project_id" validate:"required"`
+	//ServiceAccountBase64 takes precedence over  ServiceAccountJSON field
```

Comment:
```suggestion
	// ServiceAccountBase64 takes precedence over ServiceAccountJSON field
```

_(whitespace formatting)_